Source/Scene/Expression.js

```diff
@@ -47,15 +47,20 @@ define([
 
     var unaryFunctions = {
         abs : Math.abs,
-        sqrt : Math.sqrt,
-        cos : Math.cos,
-        sin : Math.sin,
-        tan : Math.tan,
         acos : Math.acos,
         asin : Math.asin,
         atan : Math.atan,
+        cos : Math.cos,
+        degrees : CesiumMath.toDegrees,
         radians : CesiumMath.toRadians,
-        degrees : CesiumMath.toDegrees
+        sin : Math.sin,
+        sqrt : Math.sqrt,
+        tan : Math.tan
```

Comment:
This may need a bit organization just to group cos/sin/tan together and near the acos/asin/atan.